api/controllers/Cid_EntryController.js

```diff
@@ -26,6 +26,11 @@ module.exports = {
       }
     ),
 
+  prebookingProcess: movements =>
```

Comment:
this is consistent with the other controllers. So happy to update however, then I would suggest that all the other controllers follow suit.